scimath/StatsFramework/ClassicalStatistics.tcc

```diff
@@ -1697,10 +1698,12 @@ std::vector<std::map<uInt64, AccumType> > ClassicalStatistics<CASA_STATP>::_data
                 // reconfigure bins and try again. This happens very rarely.
                 // See comment in _createDataArrays() at the source of the issue 
                 ThrowIf(loopCount == maxLoopCount, "Tried 5 times, giving up");
+                // increase nBins by an irrational multiplier slightly greater than 1
+                uInt64 nBinsPrev = nBins;
+                nBins *= (C::pi/3.0);
                 LogIO log;
                 log << LogIO::WARN << "Accounting error, decrease number of bins from "
```

Comment:
Shouldn't this message be changed, since nBins might now be greater than nPrevBins?